### DISCUSSION TITLE
socks5 proxy support

### DISCUSSION THREAD

**Discussion by terminaldweller** [lrstanley/girc#68] [Q&A] [answered]
does girc support socks5 proxies? I tried passing socks5 with `HTTPS_PROXY` and `ALL_PROXY` but it did not work.

**Comment by yunginnanet** [comment, marked as answer]
https://github.com/lrstanley/girc/blob/fcd9a863765616b2717560db644b9a274002f00a/conn.go#L220-L231

Yes :)
